recipes/drwav/all/conandata.yml

```diff
@@ -0,0 +1,5 @@
+sources:
+  # NOTE: https://github.com/mackron/dr_libs/blob/master/dr_wav.h#L3
```

Comment:
0.13.6 has been released. Could you please update the version? Also, use the commit on the link, because master will point to the latest only.

Reply:
Thanks for the notice. Updated in 7b63a402fadc5d2467e87924157b3882f9edbecb.